client/post-editor/media-modal/edit/_style.scss

```diff
@@ -0,0 +1,47 @@
+.editor-media-modal-edit__content {
+	background: black;
```

Comment:
@robobot3000 Let's go with `darken($gray, 52%)` for now. It's the background color I used in the mockups.

FYI, We can use any percentage with the `darken` function, but we usually use increments of 10. Check out `_colors.scss`